WIP fleet management documentation (#705)
neo4j/docs-aura: modules/ROOT/pages/fleet-management/inspect-deployment.adoc

[[sm-inspect-deployment]]
= Inspect deployment
:description: This page describes inspecting the deployment.

> From the deployment card or deployment table, select the more menu *(…​)*.

Reviewer: ```suggestion
To inspect a deployment, use the more menu *(…​)* from the deployment card or deployment table.
```

> In the graph view, the same menu can be accessed by right-clicking the DBMS node. Then select *Inspect* from the menu options.

Reviewer: ```suggestion
In the graph view, the same menu can be accessed by right-clicking the DBMS node. 
```

== Overview

> Aside from the deployment name and status, the following information is displayed.

Reviewer: ```suggestion
Aside from the deployment name and status, the following information is displayed:
```

[cols="1,2"]
|===
|Property |Description

|Version
|The Neo4j version

|Edition
|The Neo4j edition

|Packaging
|The Neo4j packaging used for deploying this cluster

|Servers
|A list of servers in the cluster

|Registered by
|Email address of the user who originally registered this deployment

|===

== Token

This section contains information and actions related to the token used to authenticate the deployment with Aura.

[cols="1,2"]
|===
|Property |Description

|Status
|Whether the token is in use or not

|Registered by
|Email address of the user who originally registered this deployment

|Creation time
|The time and date when the token was created

|Expires on
|The time and date when the token expires

|Auto-rotated
|Whether the token is configured to be automatically rotated before its expiry

|===

To perform token related actions, select the token menu *(…​)*.

> * *Regenerate*—This will remove the current token and generate a new one. Any plugin using the previous token will be logged out.

Reviewer: ```suggestion
* *Regenerate* — This removes the current token and generates a new one. 
Any plugin using the previous token will be logged out.
```

> * *Release*—This option will delete the association between this deployment and the current Neo4j cluster. This allows another cluster to claim the deployment. This is especially useful when migrating to a new major Neo4j version. Note that existing topology data will be cleared.

Reviewer: ```suggestion
* *Release* — This deletes the association between this deployment and the current Neo4j cluster. 
This allows another cluster to claim the deployment which can be especially useful when migrating to a new major Neo4j version. 
Note that existing topology data will be cleared.
```
And again with the cluster, this needs to be clarified somewhere.

Reviewer 2: Will change to deployment

> * *Delete*—Selecting this option will permanently invalidate and delete the token.

Reviewer: ```suggestion
* *Delete* — Permanently invalidates and deletes the token.
```

== License

> This section contains information about Neo4j licenses detected on this cluster.

Reviewer: Again with the cluster.
